images/atomic-registry-quickstart/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM openshift/origin
+MAINTAINER Aaron Weitekamp <aweiteka@redhat.com>
+
+ADD install.sh run.sh uninstall.sh /container/bin/
+ADD registry-console-template.yaml registry-project-template.json registry-login-template.html \
+    /container/etc/origin/
+
+LABEL name="projectatomic/atomic-registry-quickstart" \
+      vendor="Project Atomic" \
+      url="https://projectatomic.io/registry" \
+      summary="Quickstart image for Atomic Registry" \
+      description="This is a quickstart image to install Atomic Registry on a single host. It is an open source enterprise registry solution based on the Origin project featuring single sign-on (SSO) user experience, a robust web interface and advanced role-based access control (RBAC)." \
+      INSTALL='docker run -it --rm \
```

Comment:
For what audience?

On Wed, Mar 16, 2016 at 12:17 PM, Aaron Weitekamp <notifications@github.com>
wrote:

> In images/atomic-registry-quickstart/Dockerfile
> <https://github.com/openshift/origin/pull/7484#discussion_r56364160>:
>
> > @@ -0,0 +1,37 @@
> > +FROM openshift/origin
> > +MAINTAINER Aaron Weitekamp <aweiteka@redhat.com>
> > +
> > +ADD install.sh run.sh uninstall.sh /container/bin/
> > +ADD registry-console-template.yaml registry-project-template.json registry-login-template.html \
> > +    /container/etc/origin/
> > +
> > +LABEL name="projectatomic/atomic-registry-quickstart" \
> > +      vendor="Project Atomic" \
> > +      url="https://projectatomic.io/registry" \
> > +      summary="Quickstart image for Atomic Registry" \
> > +      description="This is a quickstart image to install Atomic Registry on a single host. It is an open source enterprise registry solution based on the Origin project featuring single sign-on (SSO) user experience, a robust web interface and advanced role-based access control (RBAC)." \
> > +      INSTALL='docker run -it --rm \
>
> I'm suggesting this provides the cleanest packaging and install UX without
> having to clone source or curl remote scripts, etc. The goal is to provide
> a running registry with web UI in just a few steps and be able to clean up
> the install with a single command.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7484/files/38b21ec4db20e12069f2d7a392a352497dd60a55#r56364160>
>